### PR TITLE
scripts: Remove duplicated die/info/warn functions

### PR DESCRIPTION
**Pull request by jodh-intel**
Rework the scripts that used to define their own `die()`, `info()`
and/or `warn()` functions to use the standard versions defined in
`lib/common.bash`.

Fixes: #4702.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: .ci/aarch64/install_rom_aarch64.sh [hunk 1]
```diff
@@ -112,8 +112,7 @@ install_uefi_flash()
 clean_up_and_die()
 {
 	clean_up
-	echo "ERROR: $*" >&2
-	exit 1
+	die "$*"
```
Comment (wainersm) [inline .ci/aarch64/install_rom_aarch64.sh:115]: Hi @jodh-intel , don't you need to source `lib/common.bash` in this script?

**Review by cmaf — COMMENTED** [review]
@jodh-intel a bunch of the tests are failing with the following:
```
[manage_ctr_mgr.sh:79] ERROR: Docker is already installed. Please use -f flag to force new installation
```
Perhaps a missing `source` or other env variable?

File: .ci/aarch64/install_rom_aarch64.sh (hunk 1 above)
Comment (jodh-intel) [reply .ci/aarch64/install_rom_aarch64.sh:118]: Good catch - fixed.

**Comment by jodh-intel** [comment]
@cmaf - thanks. I think I've worked out what was happening - see the new 2nd commit ;)

**Review by cmaf — APPROVED** [review]
thanks @jodh-intel

**Review by fidencio — APPROVED** [review]
lgtm, thanks @jodh-intel!

**Pull request merged**